lib/player.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+require_relative 'scoring.rb'
+
+module Scrabble
+
+  class Player
+
+    attr_reader :name, :plays, :won, :play, :player_score, :tiles
+
+    # MAX_TILES = 7
+
+    def initialize(name)
+      @name = name
+      @won = false
+      @plays = []
+      @player_score = 0
+      @tiles = []
+    end
+
+    def play(word)
+      if @won == true
+        return false
+      else
+        @plays << word
+        @player_score += Scrabble::Scoring.score(word)
+        return Scrabble::Scoring.score(word)
+      end
+    end
+
+    def total_score
+      return @player_score
+    end
+
+    def won?
+      if @player_score > 100
+        @won = true
+        return true
```

Comment:
The idea of storing `@won` is interesting, but since figuring out whether we've won is relatively easy, I don't think it's worthwhile to cache this value. It would be simpler to recompute it every time, and to call `self.won?` in `play` instead of checking `@won`.